src/lib/testrail.ts

```diff
@@ -118,14 +119,14 @@ export class TestRail {
         },
         data: JSON.stringify({ results }),
       })
-      .then(response => response.data)
-      .catch(error => { 
-        console.error(error); 
-      })
+        .then(response => response.data)
+        .catch(error => {
+          console.error(error);
+        })
```

Comment:
Could also save a line here :) 

```suggestion
        .catch(error => console.error(error))
```